database/migrations/2024_12_10_014452_create_participant_healthy_marriage_introductory_survey.php

```diff
@@ -0,0 +1,56 @@
+<?php
+
+use Illuminate\Database\Migrations\Migration;
+use Illuminate\Database\Schema\Blueprint;
+use Illuminate\Support\Facades\Schema;
+
+return new class extends Migration
+{
+    /**
+     * Run the migrations.
+     */
+    public function up(): void
+    {
+        Schema::create('participant_healthy_marriage_introductory_survey', function (Blueprint $table) {
+            $table->id();
+            $table->foreignId('participant_id')->constrained();
+            $table->date('date_of_birth')->nullable();
+            $table->string('fatherhood_program');
+            $table->boolean('become_responsible_father')->default(false);
+            $table->boolean('referred')->default(false);
+            $table->boolean('court_ordered')->default(false);
+            $table->boolean('address_child_support_concerns')->default(false);
+            $table->string('other_reason_for_attendance')->nullable();
+            $table->boolean('word_of_mouth')->default(false);
+            $table->boolean('from_past_participant')->default(false);
+            $table->boolean('family_support_division')->default(false);
+            $table->boolean('prosecuting_attorney_source')->default(false);
+            $table->boolean('marketing_source')->default(false);
+            $table->boolean('organization_itself_source')->default(false);
```

Comment:
Let's do "referred_by_good_dads" here

Reply:
On second thought, let's check with Good Dads to make sure that's what they mean